relayer/relayer/src/contracts/azero.rs

```diff
@@ -38,6 +38,12 @@ pub enum AzeroContractError {
 
     #[error("Missing or invalid field")]
     MissingOrInvalidField(String),
+
+    #[error("Dry-run reverted")]
+    DryRunReverted(Result<Value, anyhow::Error>),
```

Comment:
it's a bit weird to return Result in an Err variant. Maybe ditch the `anyhow` part as it contains no useful information anyway

Reply:
Impossible to change it, the way I see it, because it's in a crate that we don't control, and certainly don't want to fork it just for that currently.

Reply:
Unless you mean to return just `anyhow::Error`? This is probably fine.